src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

```diff
@@ -77,6 +77,11 @@
  * </p>
  * <ul>
  * <li>
+ * Property {@code commentFormat} - Specify comment pattern to trigger filter to begin suppression.
```

Comment:
The description of this property is the same as the next one. I think we can maybe mention it is tied to `influenceFormat` or something like that?